docs/installation/configuration.rst

```diff
@@ -42,6 +42,25 @@ DATABASES
    `Django documentation on databases <https://docs.djangoproject.com/en/2
    .1/ref/settings/#databases>`_
 
+DEFAULT_FILE_STORAGE
+^^^^^^^^^^^^^^^^^^^^
+
+   By default, Pulp uses the local filesystem to store files. This can be configured though to
+   alternatively use `Amazon S3 <https://aws.amazon.com/s3/>`_. To use S3, set
+   ``DEFAULT_FILE_STORAGE`` to ``pulpcore.app.models.storage.S3Storage``. The default option which
+   uses the local filesystem is ``pulpcore.app.models.storage.FileSystem``.
+
+   Note that if you use S3 you should at a minimum also set ``AWS_ACCESS_KEY_ID``,
+   ``AWS_SECRET_ACCESS_KEY``, and ``AWS_STORAGE_BUCKET_NAME``. For more S3 configuratino options,
+   see the `django-storages documents <https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html>`_.
+
+MEDIA_ROOT
+^^^^^^^^^^
+
+   The location where Pulp will store files. By default this is `/var/lib/pulp/`.
+
+   If you're using S3, this will be the path within the bucket you're using.
```

Comment:
Does the user need to set this or is it set for them? If they need to take action that could be clearer.